assets/scripts/months.js

```diff
@@ -157,9 +157,21 @@ $(document).ready(function() {
     element.find('td').off('change').off('validate');
 
   	element.find('td').on('change', function (evt) {
+
```

Comment:
Is this a better spot for making the lists, once the user is finished typing and their input is valid? Why did you decide on the validate callback?

Reply:
I think a better way to do this would be using something such as 
```suggestion
    $('.eventname').on('change', function (evt, value){
      console.log(`Our current event is: ${value}`);

      //Events_list and colors_list are used to help set up the linking system.
      var events_list = $(".eventname").map(function(){return this.innerHTML;}).get();
      var colors_list = $(".colorpick").map(function(){
        var inner_html = this.innerHTML;
        var hex = `#${inner_html.split('#')[1]}`.substring(0,7);
        return hex;
      }).get();


      console.log(`Our events are: ${events_list}`);
      console.log(`Our colors are: ${colors_list}`);

    });
```

so that it is not triggered when a user types into the month's box.